ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -845,6 +852,104 @@ namespace OrdinalAxis {
             return groupPositions;
         };
 
+        /**
+         * Get index of point inside the extended ordinal positions array.
+         *
+         * @private
+         * @function Highcharts.Axis#getIndexOfPoint
+         *
+         * @param {number} val
+         *        The pixel value of a point.
+         *
+         * @param {Array<number>} [ordinalPositions]
+         *        An array of points visible in the current plot area.
+         *
+         * @param {Array<number>} [extendedOrdinalPositions]
+         *        An array of all points available on the axis
+         *        for the given data set.
+         *
+         * @return {number}
+         */
+        axisProto.getIndexOfPoint = function (
+            val: number,
+            extendedOrdinalPositions: Array<number>
+        ): number {
+            const axis = this;
+            let firstPointVal,
+                firstPointX,
+                secondPointX;
+
+            if (axis.series[0].points) {
+                firstPointVal = axis.series[0].points[0].x;
+                firstPointX = axis.series[0].points[0].plotX;
+                secondPointX = axis.series[0].points[1].plotX;
```

Comment:
what if there is no series or there is less then 2 points in this series? Is return 0 correct way of handling this?